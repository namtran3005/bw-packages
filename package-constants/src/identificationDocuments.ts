export interface IdentificationDocument {
  isoCode: string;
  passport: boolean;
  idCard: boolean;
  baFinCompliant: boolean;
}

export const identificationDocuments: IdentificationDocument[] = [
  {
    isoCode: 'EG',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'DZ',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'AR',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'AU',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'AO',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'AZ',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'BD',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'BE',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'BA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'BR',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'BG',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'BF',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CI',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CL',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'MY',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'NA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CN',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'DK',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'DE',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'EE',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'FI',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'FR',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'GH',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'GR',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'GE',
    passport: false,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'IE',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'IS',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'IT',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'JP',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CM',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'KZ',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'KE',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'KW',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CO',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'HR',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'LB',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'LV',
    passport: false,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'LI',
    passport: false,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'LT',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'LU',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'MW',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'MK',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'ML',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'MT',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'MA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'MX',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'NZ',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'NL',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'NG',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'NO',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'AT',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'PE',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'PH',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'PL',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'PT',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'XK',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'RO',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'RU',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'SE',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'CH',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'RS',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'SG',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'SK',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'SI',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'ES',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'ZA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'KR',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'TW',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'TH',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CZ',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'TR',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'UA',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'HU',
    passport: true,
    idCard: true,
    baFinCompliant: true,
  },
  {
    isoCode: 'US',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'VE',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'GB',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
  {
    isoCode: 'CY',
    passport: true,
    idCard: false,
    baFinCompliant: true,
  },
];
