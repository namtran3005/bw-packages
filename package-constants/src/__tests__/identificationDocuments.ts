import { identificationDocuments } from '../identificationDocuments';

describe('identificationDocuments EN lists', () => {
  it('should match snapshot', () => {
    expect(identificationDocuments).toMatchSnapshot();
  });
});
