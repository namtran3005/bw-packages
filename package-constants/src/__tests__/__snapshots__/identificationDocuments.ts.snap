// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`identificationDocuments EN lists should match snapshot 1`] = `
Array [
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "EG",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "DZ",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "AR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "AU",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "AO",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "AZ",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "BD",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "BE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "BA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "BR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "BG",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "BF",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CI",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CL",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "MY",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "NA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CN",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "DK",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "DE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "EE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "FI",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "FR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "GH",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "GR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "GE",
    "passport": false,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "IE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "IS",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "IT",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "JP",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CM",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "KZ",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "KE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "KW",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CO",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "HR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "LB",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "LV",
    "passport": false,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "LI",
    "passport": false,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "LT",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "LU",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "MW",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "MK",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "ML",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "MT",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "MA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "MX",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "NZ",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "NL",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "NG",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "NO",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "AT",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "PE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "PH",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "PL",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "PT",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "XK",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "RO",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "RU",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "SE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "CH",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "RS",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "SG",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "SK",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "SI",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "ES",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "ZA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "KR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "TW",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "TH",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "CZ",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "TR",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "UA",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": true,
    "isoCode": "HU",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "US",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "VE",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "GB",
    "passport": true,
  },
  Object {
    "baFinCompliant": true,
    "idCard": false,
    "isoCode": "CY",
    "passport": true,
  },
]
`;
